Voice Note/Views/MapView.swift

```diff
@@ -1,72 +0,0 @@
-import SwiftUI
```

Comment:
@Tai @darjapo Really sorry for this file. You guys have put a lot of time and effort in this file.